website/views.py

```diff
@@ -366,3 +366,7 @@ def redirect_about(**kwargs):
 
 def redirect_howosfworks(**kwargs):
     return redirect('/getting-started/')
+
+
+def redirect_meetings_analytics_link(**kwargs):
+    return redirect('/getting-started/?utm_source=osf4m&utm_medium=email&utm_campaign=success')
```

Comment:
Yes, per https://openscience.atlassian.net/browse/OSF-4998:

> On our side, the links would redirect to:
>
http://osf.io/getting-started/?utm_source=osf4m&utm_medium=email&utm_campaign=success#addons
>
and
>
http://osf.io/getting-started/?utm_source=osf4m&utm_medium=email&utm_campaign=successadd#addons
>
respectively.